Monika After Story/game/script-compliments.rpy

```diff
@@ -530,3 +530,68 @@ label mas_compliment_bestgirl:
     m 5ekbfa "But if you ask me, you made the right choice."
     m 5hubfa "...And I'll be forever grateful that you did~"
     return
+    
+    init 5 python:
+        addEvent(
+            Event(
+                persistent._mas_compliments_database,
+                eventlabel="mas_compliment_you",
+                prompt="... I'm always thinking about you!",
+                unlocked=True
+            ),
+            eventdb=store.mas_compliments.compliment_database
+        )
+
+label mas_compliment_you:
+    if not renpy.seen_label("mas_compliment_you_2"):
+        call mas_compliment_you_2
+    else:
+        call mas_compliment_you_3
+    return
+
+label mas_compliment_you_2:
+    m 1wubso "..."
+    m 1wubfb "[player]..."
+    
+    if mas_isMoniUpset(higher=True):
+        m 2hubfb "You have no idea what that means to me."
+        
+    elif mas_isMoniDis(True):    
+        m 1ekbfa "..."
+        m "Thank you."
+        
+    elif mas_isMoniBroken(True):
```

Comment:
Same as the other one

Reply:
Fixed! Although, I am unsure if one can add _ in the titles of the complements, so we'll see if it works.